ITE/5/README.adoc

```diff
@@ -0,0 +1,182 @@
+= ITE-5: Replace signature wrapper with SSL signing spec
+:source-highlighter: pygments
+:toc: preamble
+:toclevels: 2
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| ITE
+| 5
+
+| Title
+| Replace signature wrapper with SSL signing spec
+
+| Sponsor
+| link:https://github.com/santiagotorres[Santiago Torres]
+
+| Status
+| Draft 💬
+
+
+| Type
+| Standards
+
+| Created
+| 2020-09-28
+
+|===
+
+[[abstract]]
+== Abstract
+
+This link:https://github.com/in-toto/ITE[in-toto enhancement (ITE)] proposes
+removing the in-toto signature scheme requirements, and allow users to adopt the signature wrapper that better fits their ecosystem. This has the following general benefits:
+
+1. avoid mandating an ad-hoc signing specification for in-toto; and
+2. allow for implementers to propose different signing specifications.
+
+In adition, this ITE makes a recommendation to use the
+link:http://github.com/secure-systems-lab/signing-spec[SSL Signing Spec]. This
+has the following benefits over the current state:
+
+1. Avoids canonicalization for security reasons (i.e., to not parse untrusted input) 
+2. Reduces the possibility of misinterpretation of the payload. The serialized payload is encoded as a string and verified by the recipient before de-serializing.
+3. This specification also provides a backwards-compatible serialization format.
+
+[[specification]]
+== Specification
+
+The specification adopted will be the SSL signing spec, as linked above. As
+such, we defer to that document to describe the specifics of signature
```

Comment:
Does this ITE need to explain how it will work within in-toto? In particular, the intro above says that in-toto won't require a specific envelope but will merely recommend this as one of many that are supported. Does that need to be defined here, or will that be an implementation detail? (I'm not familiar with the ITE process.)

Reply:
(Merging threads): Should we instead focus this ITE purely on supporting SSLSS (we need a better name!) and talk about how that will be implemented in in-toto?

Reply:
After discussing over video, I'll add a "processing model" that discusses how to process the Envelope layer in pseudocode. That way implementations know what they should do.

Reply:
Done. Removed the wording about supporting arbitrary envelopes and added pseudocode.